docs/conf.py

```diff
@@ -47,7 +47,12 @@
     # and CI builds https://github.com/pydata/pydata-sphinx-theme/pull/386
     "collapse_navigation": not is_release_build,
     "show_prev_next": False,
-    "native_site": False
+    # Determines the type of links produced in the navigation header:
+    # - absolute: Links point to the URL https://matplotlib.org/...
+    # - relative: Links point to top-level of the server /...
```

Comment:
Discussed with Elliott on a a call, suggested this be rename to "server_stable" or something similar..